deps/couchbeam-0.8.0/src/couchbeam_json_stream.erl

```diff
@@ -333,6 +333,10 @@ toke_string(DF, <<$\\, $u, A, B, C, D, Rest/binary>>, Acc)
             toke_string(DF, Rest, lists:reverse(Chars) ++ Acc);
         _ -> toke_string(DF, Rest, [<<16#fffd/utf8>>] ++ Acc)
     end;
+toke_string(DF, <<$\\, $u, Rest/binary>>, Acc)
+        when byte_size(Rest) < 4 ->
+    {Data, DF2} = must_df(DF, bad_escape_utf8_character_code),
+    toke_string(DF2, <<$\\, $u, Rest/binary, Data/binary>>, Acc);
 toke_string(DF, <<$\\>>, Acc) ->
```

Comment:
@Dinkor wouldn't this be treated by line 340 ?
the 2 previous clauses take \u041e\u0441 or \u041e leaving the remaining \u04
what am i missing ?

fiy, the referenced patch was an ugly one because of a thing called emoji. i tested it with cyrilic , chinese and japanese characters and all was good. obviously something went wrong, sorry for that.

this version of couchbeam is too old for bringing up to the most recent (1.6) that takes care of this. we are not updating couchbeam since we will follow a different path.